server/methods/accounts/accounts.app-test.js

```diff
@@ -431,149 +362,250 @@ describe("Account Meteor method ", function () {
+
+      callDescribed({ email });
+
+      expect(sendEmailSpy).to.have.been.called;
+      expect(createUserSpy).to.have.been.called;
+      expect(subjectSpy)
+        .to.have.been.calledWith(sinon.match(/inviteNewShopMember/));
     });
   });
 
   describe("accounts/inviteShopOwner", function () {
+    let createUserSpy;
+    let sendEmailSpy;
+    let groupId;
+    let group;
+
+    function callDescribed(accountAttributes = {}) {
+      const options = Object.assign({
+        email: fakeUser.emails[0].address,
+        name: fakeAccount.profile.addressBook[0].fullName
+      }, accountAttributes);
+
+      return Meteor.call("accounts/inviteShopOwner", options);
+    }
+
+    function stubPermissioning(settings) {
+      const { hasPermission } = settings;
+
+      sandbox
+        .stub(Reaction, "hasPermission", () => hasPermission)
+        .withArgs("admin", fakeAccount.userId, sinon.match.string);
+
+      // the following stub is just to speed things up. the tests were timing
+      // out in the shop creation step. this seems to resolve that.
+      sandbox.stub(Reaction, "insertPackagesForShop");
+    }
+
     beforeEach(function () {
-      sandbox.stub(Meteor, "user", function () {
-        return fakeUser;
-      });
+      // fakeAccount = Factory.create("account");
+      createUserSpy = sandbox.spy(MeteorAccounts, "createUser");
+      sendEmailSpy = sandbox.stub(Reaction.Email, "send");
+
+      // resolves issues with the onCreateUser event handler
+      groupId = Random.id();
+      group = Factory.create("group");
+      sandbox
+        .stub(Groups, "findOne", () => group)
+        .withArgs({ _id: groupId, shopId: sinon.match.string });
+
+      // since we expect a note to be written, let's ignore it to keep the output clean
+      sandbox.stub(Logger, "info").withArgs(sinon.match(/Created shop/));
     });
 
-    it("should ensure only admin can invite as shop owner", function () {
-      sandbox.stub(Reaction, "hasPermission", () => false);
-      const createUserSpy = sandbox.spy(MeteorAccount, "createUser");
-      expect(() =>
-        Meteor.call("accounts/inviteShopOwner", {
-          email: fakeUser.emails[0].address,
-          name: fakeUser.profile.addressBook[0].fullName
-        })).to.throw(Meteor.Error, /Access denied/);
+    it("requires admin permission", function () {
+      stubPermissioning({ hasPermission: false });
+
+      expect(callDescribed).to.throw(Meteor.Error, /Access denied/);
       expect(createUserSpy).to.not.have.been.called;
     });
 
-    it("should confirm if email already exists before creating", function (done) {
-      sandbox.stub(Reaction, "hasPermission", () => true);
-      expect(() =>
-        Meteor.call("accounts/inviteShopOwner", {
-          email: fakeUser.emails[0].address,
-          name: fakeUser.profile.addressBook[0].fullName
-        })).to.not.throw(Meteor.Error, /Access denied/);
+    it("invites existing users", function () {
+      this.timeout(1000);
```

Comment:
Isn't this less than the default?

Reply:
oops, didn't mean to commit this. the test was failing on Circle CI, but not on local, so I shortened the timeout to mimic